.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Create Zipapp and Release
+
+on:
+    push:
+        tags:
+            - "v*"
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Set up Python 3.9
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.9
+
+            - name: Install dependencies
+              run: |
+                  python -m pip install --upgrade pip
+                  pip install shiv
+
+            - name: Create Zipapp
+              run: shiv -o cloudsmith-cli.pyz -e cloudsmith_cli.cli.commands.main:main .
```

Comment:
do we need to manually specify the entrypoint here? can we just `-c cloudsmith` instead?